plugins/git-commit/git-commit.plugin.zsh

```diff
@@ -1,6 +1,6 @@
 function _git_commit_register {
   if ! git config --global --get-all alias.$1 >/dev/null 2>&1; then
-    git config --global alias.$1 '!a() { if [[ "$1" == "-s" || "$1" == "--scope" ]]; then git commit -m "'$1'(${2}): ${@:3}"; else git commit -m "'$1': ${@}"; fi }; a'
+    git config --global alias.$1 '!a() { if [ "$1" = "-s" ] || [ "$1" = "--scope" ]; then local scope="$2"; shift 2; git commit -m "'$1'(${scope}): ${@}"; else git commit -m "'$1': ${@}"; fi }; a'
```

Comment:
The use of `$1` after `shift 2` is not going to work, it should also be captured first, like `$scope`. Can you fix that?

Reply:
Nevermind, misread the code. `$1` is the one from `_git_commit_register`. LGTM